src/content/docs/es/guides/dingo/002-quick-start-overview.md

```
@@ -1,390 +1,184 @@
-
-<br>
-
-⚠️ Ajusta la ruta del enlace a la ruta correcta para la version que deseas descargar.
-
-> 💡 Consejo: Puedes descargar la ultima version de Dingo desde la pagina <a href="https://github.com/blinklabs-io/dingo/releases" target="_blank">https://github.com/blinklabs-io/dingo/releases</a>.
-
-
-Primero crearemos un directorio `dingo` para todos nuestros archivos relacionados con dingo antes de descargar el binario de Dingo.
+⚠️ Ajusta la versión y la arquitectura para que coincidan con tu sistema.
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)